extension.ts

```diff
@@ -265,6 +265,8 @@ export async function activate(context: vscode.ExtensionContext) {
     }
   }
 
+  CommandLine.SetHistoryDirPath(context.extensionPath);
```

Comment:
Also, I'm not entirely sure how I feel about this, if only for the selfish reason that it clutters up the development folder.

Nothing wrong with adding it to the .gitignore, but is there a better place to put this?

Reply:
As it's stored using the extension path which should be shared across vscode instances. we would effectively have one global history across workspaces/projects. do we want that?

Reply:
I think so. That's what vim seems to do, at least.

Reply:
vim puts. viminfo in the home directory.
If so, is it better to put this history file in your home directory?
e.g. "~/.cmdline_history", "~/.VSCodeVim/.cmdline_history"

Reply:
My vote is `~/.VSCodeVim/.cmdline_history`.

Reply:
For now, I set the file location to \~/.VSCodeVim/.cmdline_history.
a59b00aae406dcb6758d9841f433fb877714d4a0